test/loader/serializeTypeTest.cpp

```diff
@@ -361,4 +361,182 @@ TEST(serializeTypeTest, SerializeFuncRefType) {
   };
   EXPECT_EQ(Output, Expected);
 }
+
+TEST(serializeTypeTest, SerializeGCType) {
+  std::vector<uint8_t> Output;
+  std::vector<uint8_t> Expected;
+
+  // 6. Test serialize GC heap types.
+  //
+  //   1. Test NullFuncRef type.
```

Comment:
You only added the abstract heap type with `ref.null` cases.
Add some tests about the following case for example:

```
GlobalType.setValType(WasmEdge::TypeCode::Ref, WasmEdge::TypeCode::StructRef);
// This will output `0x64U, 0x6BU` (ref struct)

GlobalType.setValType(WasmEdge::TypeCode::RefNull, WasmEdge::TypeCode::StructRef);
// This will output `0x6BU` (ref.null struct), 
// which is the same as `setValType(WasmEdge::TypeCode::StructRef)`
// to prove that your code outputed the shorten reftype.
```